mantidproject/mantid
# Integrating FDA

Here the frequency domain analysis has had the new muon GUI integrated into it. This does not address the points in issue #24748. 

This PR is to be merged after ~~#24715~~ 

**Report to:** [user name]/[nobody]. 

**To test:**
Add frequency_domain_analysis_2.py to the preferences file. 

When you open mantid make sure that the original GUI still works (load data with muon analysis and then open frequency domain analysis)

Open the new GUI and check it can produce FFT's and maxents. You can compare to the results from the original GUI. 


Fixes #24734. 



Does not need to be in release notes
---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
